app/views/layouts/_empty_list.html.erb

```diff
@@ -0,0 +1,6 @@
+<div style="margin-top: 32px; border: 2px solid #b1b4b6; min-height: 300px">
```

Comment:
Could be `.app-empty-list`. Also instead of 32px, you can use the nhsuk-spacing helper

Reply:
@tvararu I've made the suggested changes in [a328455](https://github.com/nhsuk/record-childrens-vaccinations/pull/273/commits/a328455b389d8051ffb4072351a3a5fc808d3017)